core/src/Kokkos_Macros.hpp

```diff
@@ -450,6 +454,10 @@
 define KOKKOS_FORCEINLINE_FUNCTION inline
 #endif
 
+#if !defined(KOKKOS_IMPL_FORCEINLINE)
+    define KOKKOS_IMPL_FORCEINLINE inline
```

Comment:
??? :(

Reply:
Yeah I don't know what happened here.
```suggestion
    #define KOKKOS_IMPL_FORCEINLINE inline
```